test/integration/load/load_test.go

```diff
@@ -218,8 +260,9 @@ func TestValidCNSStateDuringScaleAndCNSRestartToTriggerDropgzInstall(t *testing.
 	require.NoError(t, err)
 
 	// Validate the CNS state
-	err = validator.Validate(ctx)
-	require.NoError(t, err)
+	if testConfig.ValidateStateFile {
```

Comment:
Thanks for catching this. We should be validating here no matter what. Otherwise the test is not complete. 

Considering that this test requires validating the statefile I will revert 219 as well so we have before, during, and after results.